scripts/testnet/sovereignBridge/pyScripts/update_toml.py

```diff
@@ -115,20 +117,23 @@ def main():
     main_chain_address = sys.argv[1]
     sovereign_chain_address = sys.argv[2]
     esdt_prefix = sys.argv[3]
-    main_chain_elastic = sys.argv[4]
+    use_elasticsearch = sys.argv[4]
+    main_chain_elastic = sys.argv[5]
```

Comment:
[optional]

Could've made a simpler logic here to decide if boolean flag `use_elasticsearch` is true based on len(main_chain_elastic). E.g: if empty url, then no use

Reply:
main_chain_elastic always has value because is a constant in `config/configs.cfg`

Reply:
The value of that config could be empty